lib/index.js

```diff
@@ -14,7 +14,8 @@ var internals = {
 };
 
 exports.settings = {
-    truncateMessages: true
+    truncateMessages: true,
+    prototype: true
```

Comment:
I'm not huge on using the name `prototype`, since these settings apply to the entire module, not just `deep.equal()`. I don't really have a better idea except maybe `comparePrototypes`.